src/main/java/org/jabref/logic/importer/EntryBasedParserFetcher.java

```diff
@@ -69,4 +52,21 @@ default void doPostCleanup(BibEntry entry) {
             throw new FetcherException("An internal parser error occurred", e);
         }
     }
+
+    /**
```

Comment:
I think, the location in the code can be kept --> `git checkoupt upstream/master -- src/main/java/org/jabref/logic/importer/EntryBasedParserFetcher.java`